client/src/components/AddProduct.js

```diff
@@ -0,0 +1,249 @@
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  paper: {
+    position: "absolute",
+    display: "flex",
+    //display:'block',
+    flexDirection: "column",
+    alignItems: "center",
+    //backgroundColor: theme.palette.background.paper,
+    backgroundColor: "#f8f8f8",
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+    //padding: theme.spacing(1, 1, 3),
+    borderRadius: "0.3rem",
+    overflow: "hidden",
+    outline: "none",
+    minWidth:"30rem",
+    maxHeight: "41rem",
+    height:'75%',
+  },
+  addButton: {
+    borderRadius: "10rem",
+    padding: theme.spacing(2, 6),
+    backgroundColor: "#DF1B1B",
+    color: "white",
+    margin: theme.spacing(2),
+    width: "11rem",
+    height: "3.5rem",
+    fontSize: "0.8rem"
+  },
+  titleContainer: {
+    display: "flex",
+    //padding: theme.spacing(0, 0, 3),
+    maxHeight:"3rem"
+  },
+  closeButtonContainer: {
+    textAlign: "right",
+    width: "100%"
+  },
+  titleText: {
+    fontSize: "1.5rem",
+    fontWeight:"bold"
+  },
+  itemText:{
+    fontSize: "0.8rem",
+    fontWeight: "400",
+    letterSpacing:"0.5px"
+  },
+  productCard: {
+    display: "flex",
+    flexDirection: "row",
+    maxWidth: "30rem"
+  },
+  removeButtonContainer:{
+    display: "flex",
+    alignItems:"center",
+    justifyContent:"center",
+  },
+  removeButton:{
+    display: "flex",
+    alignItems:"center",
+    justifyContent:"center",
+    borderRadius: "10rem",
+    height:"3rem",
+  },
+  addButtonContainer:{
+    display: "flex",
+    justifyContent:"center",
+    margin:theme.spacing(3,0,2)
+  },
+  bodyContainer:{
+    minWidth:"30rem",
+    maxHeight: "29rem",
+    overflow:"hidden",
+    display:"flex",
+    justifyContent:"center"
+  },
+  bodyContent:{
+    minWidth:"30rem",
+    maxHeight: "30rem",
+    overflow:"scroll",
+    overflowX:"hidden",
+  },
+  textFieldContainer:{
+    display:"flex",
+    justifyContent:"center",
+    width:"80%",
+    backgroundColor:"white"
+  },
+  centerText:{
+    textAlign:"center"
+  },
+  imageFieldContainer:{
+    display:"flex",
+    backgroundColor:"white",
+    width:"35%",
+    height:"0",
+    paddingBottom:"35%"
+  },
+  selectListInput: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "center",
+    backgroundColor: "white",
+    margin: "3rem",
+    overflow: "hidden",
+    height: "4rem",
+    padding: "0 0.5rem",
+    boxShadow: "0 0 0.5rem 1px #eee",
+    width: "80%"
+  }
+}));
+
+const AddList = (props) => {
+    const {listTitle, listOpen, changeListOpen, addProductOpen, changeAddProductOpen, demoListsArray} = props;
+    const classes = useStyles()
+    const [title, setTitle]=useState('');
+    const [list, setList]=useState('');
+    const [errors, setErrors]=useState({});
+
+    const handleChange = (event) => {
+      setList(event.target.value);
+    };
+
+    const handleClose = () => {
+        changeAddProductOpen();
```

Comment:
Looks like indents aren't consistent. Can you check your prettier settings?

Reply:
It wasn't on when I initially made this, thanks!